### DISCUSSION TITLE
Wifi connection

### DISCUSSION THREAD

**Discussion by thebongmaster** [openbouffalo/buildroot_bouffalo#48] [Q&A]
Hi, just got this installed on my 0x64, but cannot find any info on how to connect to wifi network :x

**Comment by gamelaster** [comment]
No drivers

**Reply by thebongmaster** [reply]
ahh poop. hope they arrive soon :)

**Reply by gamelaster** [reply]
@thebongmaster we are waiting for Bouffalo to release binary blobs and some documentation, so we can implement it.

**Reply by thebongmaster** [reply]
so there is no network connectivity at all on this yet? i looked at the projects page and there seems alot waiting on them blobs X3 i got 2 of these boards so will try other things on the other one and reserve the current one for playing with this linux :)
